style.css

```diff
@@ -7,7 +7,7 @@ main {
   position: relative;
   width: 375px;
   height: 5315px;
-  background: #f7f7f9;
+  background: #e5e5e5;
```

Comment:
- [ Optional] I don't know why you changed the background to this color but it is not the color used in the Figma design, kindly consider changing it your project will look great.